### DISCUSSION TITLE
NODATA value

### DISCUSSION THREAD

**Discussion by WCMC-vblanque** [isciences/exactextract#102] [Q&A]
Is there a way to know how much nodata pixels (partial or total) was inside my polygon.

I am in a case, where almost half of the pixels inside the polygon of interest are nodata values. I would like to be transparent when I publish the zonal stats to say to the end user that these stats are only considering half of the polygon.

I don't understand if this is already possible with exactextract? Any parameter related to ignore or consider nodata value or counting the number of pixels with nodata values?

**Comment by tm-jc-nacpil** [comment]
Hi @WCMC-vblanque @dbaston ! I am also following this thread. It would be useful for us to know how NODATA values are considered and if the NODATA parameter can be set by the user. For example, some rasters might have NODATA values that are not properly specified in the metadata and we usually have to set them manually when using other zonal statistics tools such as `rasterstats`

**Reply by sam-bradshaw-wcmc** [reply]
This issue may address your question https://github.com/isciences/exactextract/issues/100

**Reply by tm-jc-nacpil** [reply]
Hi @sam-bradshaw-wcmc thanks for this! I'll check it out :D
